src/daily-quest/suggestion/suggestion.types.ts

```diff
@@ -1,7 +1,23 @@
-export default interface DailyQuestSuggestionDTO {
+import { DailyQuestSuggestionModel } from "@/daily-quest/suggestion/suggestion.domain.types";
+
+export interface DailyQuestSuggestionResponse {
   id: string;
   title: string;
   icon: string;
   created_at: string;
   updated_at: string;
 }
+
+export const toDailyQuestSuggestionResponse = (
+  model: DailyQuestSuggestionModel
+): DailyQuestSuggestionResponse => ({
+  id: model.id,
+  title: model.title,
+  icon: model.icon,
+  created_at: model.created_at.toISOString(),
+  updated_at: model.updated_at.toISOString(),
+});
+
+export const toDailyQuestSuggestionResponses = (
```

Comment:
```suggestion
export const toDailyQuestSuggestionResponseList = (
```

Reply:
Nice idea! Fixed for all entities. Thanks!